app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RatingsController < ApplicationController
+  def new
+    @rating = Rating.new
+    @user_id = params['user_id']
+  end
+
+  def create
+    Rating.create(user_id: params["user_id"], comment: params["rating"]["comment"])
```

Comment:
Setup Artisan and Rating has_many/belongs_to association and refactor the new and create actions.